freedom-openocdcfg-generator.c++

```diff
@@ -393,11 +393,17 @@ static void show_dts_memory(void) {
   std::cout << "}" << std::endl;
 }
 
-static void write_config_file(fstream &os, std::string board) {
+static void write_config_file(fstream &os, std::string board,
+                              std::string protocol) {
   os << "#" << __FUNCTION__ << std::endl;
   os << "# JTAG adapter setup" << std::endl;
   os << "adapter_khz     10000" << std::endl << std::endl;
 
+  if (protocol == "cjtag") {
+    os << "#source [find interface/ftdi/olimex-arm-jtag-cjtag.cfg]" << std::endl
+       << std::endl;
+  }
+
```

Comment:
This assume that OpenOCD package has been downloaded and installed.
It would not be an issue for Freedom Studio, but for Freedom-E-SDK command line it is not necessary openocd get download and installed before BSP get generated.

Reply:
If OpenOCD hasn't been installed it won't work anyway 🤷‍♂